openfl-tutorials/interactive_api/PyTorch_TinyImageNet_XPU/README.md

```diff
@@ -0,0 +1,84 @@
+# PyTorch_TinyImageNet
+
+## **How to run this tutorial (without TLC and locally as a simulation):**
+<br/>
+
+### 0a. If you haven't done so already, create a virtual environment, install OpenFL, and upgrade pip:
+  - For help with this step, visit the "Install the Package" section of the [OpenFL installation instructions](https://openfl.readthedocs.io/en/latest/install.html#install-the-package).
+
+<br/>
+
+### 0b. Quick XPU Setup
```

Comment:
You should also mention the hardware prerequisite here (Intel® Data Center GPU Max Series)

Reply:
Added the hardware prerequisite